src/Microsoft.AspNetCore.Razor/Parser/TagHelpers/Internal/TagHelperParseTreeRewriter.cs

```diff
@@ -752,7 +752,18 @@ private static string GetTagName(Block tagBlock)
             }
 
             var childSpan = (Span)child;
-            var textSymbol = childSpan.Symbols.FirstHtmlSymbolAs(HtmlSymbolType.WhiteSpace | HtmlSymbolType.Text);
```

Comment:
Apparently nothing else uses the extension method. I'll remove the type instead.